### ISSUE TITLE
Report saved in wrong folder

### ISSUE THREAD

**Issue by gscocozza-southworks** [BBE78/cypress-sonarqube-reporter#6]
**Describe the bug**
i have got this config but in the moment as i set ` "useAbsoluteSpecPath": true` the report was saved next to the test files instead of the specified fodler
```
{
  "baseUrl": "http://localhost:3000/",
  "reporter": "cypress-sonarqube-reporter",
  "video": false,
  "screenshotOnRunFailure": false,
  "reporterOptions": {
    "outputDir": "./cypress-report",
    "overwrite": true,
    "useAbsoluteSpecPath": true
  }
}
```

**Screenshots**
This shouldn't happend
![imagen](https://user-images.githubusercontent.com/61709292/99257842-6da68600-27f6-11eb-98d1-0c71d2091772.png)

**Desktop (please complete the following information):**
-   OS: \Windows 10 x64 v1909
-   Browser \Electron
-   Version \IDK

**Comment by BBE78** [comment]
Hi @gscocozza-southworks, I confirm the issue. Correction is in progress...

**Comment by BBE78** [comment]
Fixed in v1.7.0
